pandas/io/tests/test_json/test_pandas.py

```diff
@@ -178,7 +178,11 @@ def _check_orient(df, orient, dtype=None, numpy=False,
                 self.assertTrue(df.columns.equals(unser.columns))
             elif orient == "values":
                 # index and cols are not captured in this orientation
-                assert_almost_equal(df.values, unser.values)
+
+                if df.shape == (0, 0):
+                    assert unser.shape[0] == 0
```

Comment:
The original test now raises assertion because of array shape is being tested. I couldn't distinguish the below behavior is classified into a bug.

```
df = pd.DataFrame([])
df
# Empty DataFrame
# Columns: []
# Index: []

# OK
pd.read_json(df.to_json(orient='values'), orient='values')
# Empty DataFrame
# Columns: []
# Index: []

# NG column name "0" is added.
pd.read_json(df.to_json(orient='values'), orient='values', numpy=True)
# Empty DataFrame
# Columns: [0]
# Index: []
```